test/utils/utils.test.js

```diff
@@ -200,7 +200,23 @@ describe('Utils', () => {
     });
   });
 
-  describe ('rtlSupport', () => {
+  describe('video uploaded using franklin bot', () => {
+    it('decorates video', async () => {
+      const block = document.createElement('div');
+      const a = document.createElement('a');
+      const href = 'https://main--blog--adobecom.hlx.page/media_17927691d22fe4e1bd058e94762a224fdc57ebb7b.mp4';
+      a.setAttribute('href', href);
+      a.textContent = href;
+      block.append(a);
```

Comment:
You likely don't need to create your block on the fly like this since line 13 sets the mock you edited to the innerHTML of the document: 
`
document.body.innerHTML = await readFile({ path: './mocks/body.html' })
`

Reply:
The mock video `a` tag in body.html gets decorated on line 147 as
`loadArea` ->  `decorateSections` -> `decorateLinks` -> `decorateVideo`. 
To mock that again, will need to import the original `body.html`.

Reply:
Ah, right you are! Very cool.

Seems to speak to the need to break some of these tests up, but that is beyond the scope of this ticket.